clis/tcli/src/bin/tclid/main.rs

```diff
@@ -0,0 +1,189 @@
+#![deny(missing_docs)]
+// TCLID doc rs documentation introduction:
+#![doc = include_str!["./doc/tclid_documentation.md"]]
+
+//! Main and code for the API which is used for sending commands to libtelio which runs on a daemon.
+
+use std::path::PathBuf;
+
+use anyhow::{anyhow, Result};
+use clap::Parser;
+use tokio::time::Duration;
+
+mod coms;
+mod daemon;
+mod tclid;
+mod telio_cli_interface;
+
+use coms::DaemonIPC;
+use daemon::{Daemon, ProcessType};
+use telio_cli_interface::TelioCliInterface;
+
+/// Name for the local socket file which will be used for inter process communication.
+const DAEMON_IPC_SOCKET: &str = "tclid.sock";
+/// Path where all the logs, local socket and pid file will be stored, relative to the path of this binary.
+const TCLID_WD_PATH: &str = "/var/run/tclid_wd";
```

Comment:
:thinking: This may be not portable :/.

There is this (runtime_dirs)[https://docs.rs/dirs/latest/dirs/fn.runtime_dir.html] thing, which sounds like a perferct candidate for our purposes, but, unfortunately, it is only supported for Linux, so we would not gain any compatability. 

As a compromise, we _could_ use (dirs::data_dir())[https://docs.rs/dirs/latest/dirs/fn.data_dir.html] which is not ideal, but at least portable.

Reply:
Because Windows is not supported right now and adding support for it would require a lot of work, we've agreed to leave this as it is, because on Unix this is the standard location for pid and sock files.